roles/openshift_loadbalancer/templates/haproxy.docker.service.j2

```diff
@@ -1,7 +1,7 @@
 [Unit]
-After={{ openshift.docker.service_name }}.service
-Requires={{ openshift.docker.service_name }}.service
-PartOf={{ openshift.docker.service_name }}.service
+After={{ openshift_docker_service_name }}.service
```

Comment:
openshift_docker_service_name is one variable that needs to have a fact set.  This is because we cannot assume 'docker' here, and the user does not set a variable directly for this value.